components/elements/cards/cardevent.stories.js

```diff
@@ -0,0 +1,57 @@
+import { withA11y } from '@storybook/addon-a11y'
+import { withKnobs } from '@storybook/addon-knobs'
+
+import {
+  MtlCardevent,
+  MtlButton,
+  Mtlparagraph,
+  MtlTextinfo,
+  IconCalendarSharp,
+  IconTimeOutline,
+  MtlH4
+} from '@/components/index'
+
+export const CardEvent = () => ({
```

Comment:
About the file name:
I saw an `inputs/text-input.stories.js`
And this one is: `/cards/cardjob.stories.js `
So it should be: `/cards/card-job.stories.js ` and maybe: `/inputs/input-text.stories.js `
I let you judget